samples/word/50-document/manage-settings.yaml

```diff
@@ -0,0 +1,133 @@
+order: 6
+id: word-document-manage-settings
+name: Manage settings
+description: 'This sample shows how to add, get, and delete settings in the file.'
```

Comment:
Is it worth defining "settings"? To me, settings are like configurations (zoom, ribbon locking, etc). What's the user scenario?